far2l/src/cfg/config.cpp

```diff
@@ -380,6 +386,9 @@ static LONG_PTR WINAPI DirCfgDlgProc(HANDLE hDlg, int Msg, int Param1, LONG_PTR
 		SendDlgMessage(hDlg, DM_LISTSET, ID_DIRCFG_STYLE_COMBO, (LONG_PTR)&farlist);
 //		SendDlgMessage(hDlg, DM_LISTSETCURPOS, dircfg_data->DirNameStyleComboID, (LONG_PTR)&flpos);
 		update_surrcombo( );
+
+		swprintf(tmp, 48, L"\"%ls\"", SymLinkNames[dircfg_data->DirNameStyle].CPtr());
```

Comment:
please use ARRAYSIZE(tmp) instead of hardcoded 48 (btw its 48 * 4)

Reply:
Done